overcooked_ai_py/mdp/overcooked_env.py

```diff
@@ -238,26 +252,27 @@ class Overcooked(gym.Env):
     def custom_init(self, base_env, featurize_fn, baselines=False):
         """
         base_env: OvercookedEnv
-        featurize_fn: what function is used to featurize states returned in the 'both_agent_obs' field
+        featurize_fn(mdp, state): fn used to featurize states returned in the 'both_agent_obs' field
         """
         if baselines:
             # NOTE: To prevent the randomness of choosing agent indexes
             # from leaking when using subprocess-vec-env in baselines (which 
             # seeding does not) reach, we set the same seed internally to all
             # environments. The effect is negligible, as all other randomness
             # is controlled by the actual run seeds
-            np.random.seed(0)
+            # np.random.seed(0)
+            pass
```

Comment:
... Should this entire if block and comment be deleted?

Reply:
I feel like it was non-obvious (at least to me) that if using parallel threads it might be harder to propagate random seeds to the threads, but I guess you're right in that this is possibly baselines specific and shouldn't be in this repo.

Reply:
Oh I'm just saying that you currently have `if <condition>: pass` which is odd. I don't really understand what the problem is, why this used to be there, why it isn't there now, how other people are supposed to interact with this code; I'm just flagging it as a thing that's confusing me.